### PR TITLE
Adds file and npm scripts to convert/parse ArcGIS or GEOJson files.

### PR DESCRIPTION
**Pull request by gseyffert**

### CODE REVIEW

**Comment by jgravois** [comment]
very cool! my first instinct is that we should isolate this as an `/example` with its own package.json which declares its dependencies instead of putting it in the root of the project and shipping it every time someone calls `npm install terraformer-arcgis-parser`.

that said, im stoked to see the contribution. i'll make time to run it myself and give more feedback shortly.

File: examples/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "keywords": [
+    "ArcGIS",
+    "Esri",
+    "GIS",
+    "Geography"
+  ],
+  "author": "Graham Seyffert <gseyffert@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "terraformer-arcgis-parser": "~1.0.5"
```
Comment (jgravois) [inline examples/package.json]: and terraformer core...

File: examples/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "git@github.com:Esri/terraformer-arcgis-parser.git"
+  },
+  "keywords": [
+    "ArcGIS",
+    "Esri",
+    "GIS",
+    "Geography"
+  ],
+  "author": "Graham Seyffert <gseyffert@gmail.com>",
+  "license": "MIT",
```
Comment (jgravois) [inline examples/package.json]: Apache-2.0

File: examples/package.json (hunk 1 above)
Comment (gseyffert) [inline examples/package.json]: More than happy to add it explicitly but fwiw npm install grabbed both dependencies for me since parser relies on core.

Comment (jgravois) [inline examples/package.json]: makes sense. cool!

**Comment by jgravois** [comment]
finally got a chance to test this today.  worked like a charm with a 100MB .geojson input

thanks for your contribution @gseyffert!

**Comment by jgravois** [comment]
i pushed a README for the new sample in 2dfaf210e59ed93abc173b0b030956e0bc6f8c86

**Pull request merged**